### PR TITLE
Fix clusterer bug

### LINKED ISSUE #101: Placemark в Clusterer не отображаются
Привет всем.

Почему-то не отображается Placemark в Clusterer.
Сделано по примеру https://gribnoysup.github.io/react-yandex-maps/#/sandbox/clustering/clusterer-create , но удаляя <Clusterer> как родителя - булавки рендерятся. Добавляю <Clusterer> - ничего нет. 

В консоли ошибок нет. В чем может быть проблема? Спасибо :(

v2.2.1

### PR DESCRIPTION
**Pull request by mmarkelov**
Fixes #101 
According to react 16.5.0 **noscript** tag will be ignored

### CODE REVIEW

**Comment by mmarkelov** [comment]
Checked it in [sandbox](https://codesandbox.io/s/l7oqnz7mml)

File: src/Clusterer.js [hunk 1]
```diff
@@ -115,9 +115,9 @@ export class Clusterer extends React.Component {
     const { instance } = this.state;
 
     return (
-      <noscript>
+      <div>
```
Comment (gribnoysup) [inline src/Clusterer.js]: @mmarkelov can you apply the same change to Control element, please? I would do it myself, but this branch is in your fork and I don't have access to it 😅

Comment (mmarkelov) [reply src/Clusterer.js]: @gribnoysup sure!

Comment (mmarkelov) [reply src/Clusterer.js]: @gribnoysup I just leave div here to support react < 16

Comment (mmarkelov) [reply src/Clusterer.js]: @gribnoysup Should we place it inside Helpers.js?

**Pull request merged**